cookies/views.py

```diff
@@ -977,24 +978,48 @@ def entity_change_concept(request, entity_id):
             form = ConceptEntityLinkForm()
 
     if request.method == 'POST':
-        form = ConceptEntityLinkForm(request.POST)
-        if form.is_valid():
-            uri = form.cleaned_data.get('uri')
+        if 'save' in request.POST:
```

Comment:
What's going on in this big block of code you added? Please add some narrative in comments to explain what you're up to.